framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -1579,10 +1583,19 @@ public ParameterizedExecutableType methodFromUse(MethodInvocationTree tree) {
         if (dependentTypesHelper != null) {
             dependentTypesHelper.viewpointAdaptMethod(tree, method);
         }
-        poly.resolve(tree, method);
+
         return mType;
     }
 
+    @Override
+    public void methodFromUsePreSubstitution(ExpressionTree tree, AnnotatedTypeMirror mirror) {
+        // no call to super needed since super is no-op
```

Comment:
Well, there is a call to super on the next line... why assume that the behavior in the super method will never change?

Reply:
I will remove this line